data/remnant missions.txt

```diff
@@ -1320,6 +1320,19 @@ event "remnant: return the samples timer"
 
 
 
+# Compatibility patch for those who made completed Return the Samples 2 before 0.9.11, where it did not have the timer.
+mission "Remnant Return the Samples Compatability Patch"
```

Comment:
Compatibility

Also, remove "made" from the comment

Reply:
Fixed. Thanks!